backend/audit/etl.py

```diff
@@ -58,25 +58,19 @@ def load_findings(self):
         for entry in findings_uniform_guidance_entries:
             findings = entry["findings"]
             finding = Finding(
+                award_seq_number=entry.get("seq_number", 1),
+                report_id=self.report_id,
+                finding_seq_number=1,  # TODO: Where does this come from?
```

Comment:
Most likely it will be this value ["seq_number"](https://github.com/GSA-TTS/FAC/pull/1505/files#diff-bf46162f84e6a008a08ed3d0e697189966a10bbd12f4d2f1dacde7c72b20c60cR61)